stable/drupal/README.md

```diff
@@ -55,10 +55,10 @@ The following tables lists the configurable parameters of the Drupal chart and t
 | `mariadb.mariadbRootPassword`     | MariaDB admin password                | `nil`                                                     |
 | `serviceType`                     | Kubernetes Service type               | `LoadBalancer`                                            |
 | `persistence.enabled`             | Enable persistence using PVC          | `true`                                                    |
-| `persistence.apache.storageClass` | PVC Storage Class for Apache volume   | `nil` (uses alpha storage class annotation)               |
+| `persistence.apache.storageClass` | PVC Storage Class for Apache volume   | `nil` (uses beta storage class annotation)               |
```

Comment:
This is a little confusing because the "(uses alpha storage class annotation)" addresses the fact that the alpha annotation is used by default, whereas I think you're using this to say setting a storageClass will use the beta annotation. I think given that this is under the default column, the original text is more correct. Thoughts?